From 2a5f3fd2df06131608b2cff375baf1aa444f9366 Mon Sep 17 00:00:00 2001
From: Daniel Sheeler <dsheeler@pobox.com>
Date: Sun, 24 Mar 2019 19:47:51 -0500
Subject: [PATCH 1343/1936] Reverse master and part metering.

This is something I don't fully grok.  Maybe we need some thinking to
figure out this reversed stereo situation.
---
 src/Misc/Master.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/Misc/Master.cpp b/src/Misc/Master.cpp
index b32f77f9..b92e513c 100644
--- a/src/Misc/Master.cpp
+++ b/src/Misc/Master.cpp
@@ -976,7 +976,7 @@ void Master::setController(char chan, int type, int par)
     }
 }
 
-void Master::vuUpdate(const float *outl, const float *outr)
+void Master::vuUpdate(const float *outr, const float *outl)
 {
     //Peak computation (for vumeters)
     vu.outpeakl = 1e-12;
@@ -1009,8 +1009,8 @@ void Master::vuUpdate(const float *outl, const float *outr)
         vuoutpeakpartl[npart] = 1.0e-12f;
         vuoutpeakpartr[npart] = 1.0e-12f;
         if(part[npart]->Penabled != 0) {
-            float *outl = part[npart]->partoutl,
-            *outr = part[npart]->partoutr;
+            float *outr = part[npart]->partoutl,
+            *outl = part[npart]->partoutr;
             for(int i = 0; i < synth.buffersize; ++i) {
                 if (fabs(outl[i]) > vuoutpeakpartl[npart])
                     vuoutpeakpartl[npart] = fabs(outl[i]);
-- 
2.47.0

